### ISSUE TITLE
conch.ssh.address need to be ported to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8495]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)| @einarfd reported|
|-|-|
|Trac ID|trac#8495|
|Type|enhancement|
|Created|2016-06-23 15:09:45Z|
|Branch|https://github.com/twisted/twisted/tree/8495-einarfd-conch_ssh_address_py3|

The functionality in conch.ssh.address hasn't been ported to Python 3 yet, and is needed to get SSH to work on Python 3. 

<details><summary>Searchable metadata</summary>

```
trac-id__8495 8495
type__enhancement enhancement
reporter__einarfd einarfd
priority__normal normal
milestone__None None
branch__8495_einarfd_conch_ssh_address_py3 8495-einarfd-conch_ssh_address_py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1466694585476487 1466694585476487
changetime__1466775748315932 1466775748315932
version__None None
owner__einarfd einarfd

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)<a name="note_1"></a>|@einarfd commented|
|-|-|

Added a pull request for this at https://github.com/twisted/twisted/pull/209

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @einarfd|
|-|-|

Changes look good. Thanks!

Waiting for build results and will merge.

Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset 48f64353d13c30f867b0ae05ea345868b264bb12
```
#!CommitTicketReference repository="" revision="48f64353d13c30f867b0ae05ea345868b264bb12"
Merge 8495-einarfd-conch_ssh_address_py3: Port t.conch.ssh.address to Python 3.

Author: einarfd
Reviewer: adiroiban
Fixes: #8495
```
